carbon-design-system/carbon
# refactor(react): update FilterableMultiSelect to match WCAG 2.1 AA

Closes #7861

Update our FilterableMultiSelect component to match the changes that we made to our Combobox for WCAG 2.1 AA.

#### Changelog

**New**

**Changed**

- Update next/ListBoxSelection to accept additional props for the underlying `<button>`
- Update FilterableMultiSelect to use new structure, refactor from previous onStateChange patterns to simplify

**Removed**

#### Testing / Reviewing

- Visit the storybook to make sure the component operates as intended
- Run AC when the menu is open or closed to make sure no violations occur
